package.json

```diff
@@ -57,11 +57,10 @@
     "eslint": "^3.3.1",
     "http-server": "^0.9.0",
     "jsdom": "^9.4.2",
+    "lodash": "^4.15.0",
```

Comment:
If the implementation uses lodash then it must be a dependency, not a dev one, otherwise people installing from npm will struggle.